src/main/java/com/example/myKitchenManager/mysqldb/MySQLSBUtil.java

```diff
@@ -0,0 +1,13 @@
+package com.example.myKitchenManager.mysqldb;
+
+public class MySQLSBUtil {
```

Comment:
A better solution is that we put the DB configurations in to config files. For example, we can have a local.yaml which contain configs connect to our Local DB, a staging.yaml config that contain configs connect to our staging DB and a prod.yaml that contain configs connect to our prod DB. Then we only need to pass a environment variable when start the application it will automatically pick the right config and connect to the correspondent DB.

Reply:
Find a good resource for this question. https://stackoverflow.com/questions/34845990/spring-use-one-application-properties-for-production-and-another-for-debug